src/lib/features/playground/playground-service.ts

```diff
@@ -198,23 +199,30 @@ export class PlaygroundService {
                     const strategyEvaluationResult: FeatureStrategiesEvaluationResult =
                         client.isEnabled(feature.name, clientContext);
 
+                    const hasUnsatisfiedParent = Boolean(
+                        strategyEvaluationResult.hasUnsatisfiedParent,
+                    );
                     const isEnabled =
                         strategyEvaluationResult.result === true &&
-                        feature.enabled;
+                        feature.enabled &&
+                        !hasUnsatisfiedParent;
 
                     return {
                         isEnabled,
                         isEnabledInCurrentEnvironment: feature.enabled,
+                        hasUnsatisfiedParent,
                         strategies: {
                             result: strategyEvaluationResult.result,
                             data: strategyEvaluationResult.strategies,
                         },
                         projectId: featureProject[feature.name],
-                        variant: client.forceGetVariant(
-                            feature.name,
-                            strategyEvaluationResult,
-                            clientContext,
-                        ),
+                        variant: isEnabled
+                            ? client.forceGetVariant(
```

Comment:
since this version of a client does not check feature enabled status this code was returning variants for disabled features. This PR fixes this bug

Reply:
Wasn't this by design? "If this feature is enabled, it would return X variant"?